dev/com.ibm.ws.classloading/resources/com/ibm/ws/classloading/internal/resources/ClassLoadingServiceMessages.nlsprops

```diff
@@ -155,3 +155,7 @@ cls.classloader.config.typo.useraction=Check the server configuration for invali
 cls.classloader.config.typo2=CWWKL0083E: When using prefix + or -, [{0}] API type is not valid in this [{1}] list.  All API type must have a prefix of + or -
 cls.classloader.config.typo2.explanation=Change the invalid API type to a valid API type with + or -.
 cls.classloader.config.typo2.useraction=Check the server configuration for invalid API type to ensure that the API type are correct.
+
+cls.classloader.suggested.feature=CWWKL0084W: Unable to load class {0}, try enabling the {1} (or newer) feature in your server configuration.
```

Comment:
Change "Unable to load class {0}, try enabling the {1} (or newer) feature in your server configuration." to "The {0} class could not be loaded. Try enabling the {1} feature or a newer version of the feature in your server configuration.".

Reply:
The intent of this message is to help users migrate to Java 11 easier by offering feature suggestions. I suspect the suggestions will be correct _most_ of the time, but since they may not be correct 100% of the time I don't want to make this an Error message, which to me indicates that something is broken and must be fixed.